pkg/analyze/host_cpu.go

```diff
@@ -61,13 +62,39 @@ func (a *AnalyzeHostCPU) Analyze(
 		return []*AnalyzeResult{&result}, err
 	}
 
+	content := collectedContents[0].Data
```

Comment:
Why only check the first element? Isn't this supposed to be results per node? We should check all entries like we do in other places.

Reply:
this is just for templating the output message, it isn't used for doing the check itself I think

Reply:
that being said, let me take a look

Reply:
ugh that does make the output message much less potentially useful